--- urbackup-client-2.1.7.0/defaults_client.orig	2016-12-17 06:48:17.231723312 +0000
+++ urbackup-client-2.1.7.0/defaults_client	2016-12-17 06:48:29.152723153 +0000
@@ -1,6 +1,6 @@
 # Defaults for urbackup_client initscript
 # sourced by /etc/init.d/urbackupclientbackend
-# installed at /etc/default/urbackupclient by the maintainer scripts
+# installed at /etc/conf.d/urbackupclient by the maintainer scripts
 
 #
 # This is parsed as a key=value file
